tuning/config/tracker_configs.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+@dataclass
+class AimConfig:
+    # Name of the experiment
+    experiment: str = None
+    # 'repo' can point to a locally accessible directory (e.g., '~/.aim') or a remote repository hosted on a server.
+    # When 'remote_server_ip' or 'remote_server_port' is set, it designates a remote aim repo.
```

Comment:
when it's remote, does it mean a remote directory path?

Reply:
No. Remote here means `http://server:port` I can link to the aim documentation here but felt anyone using this would already know how aimstack works.